web/templates/support/index.html.eex

```diff
@@ -1,203 +1,151 @@
+        or find us at 217, Maryon Road, Charlton, Greenwich, SE7 8DB
+    </p> 
+
+    <p><strong><%= link "Greenwich Family Voice", to: " http://www.greenwichparentvoice.com", class: "hl-aqua flex-wrap underline" %></strong> an independent group
+      of parents in Greenwich who have a child with special educational needs or a disability. Greenwich Family Voice are committed
+      to giving a voice to all carers of disabled children in government. Email <%= link "info@greenwichparentvoice.com", to: "mailto:info@greenwichparentvoice.com", class: "hl-aqua flex-wrap underline" %> 
+      or write to Greenwich Parent Voice, PO Box 71515, London, SE10 1DH
+    </p>
 
-    <p><strong><%= link_to("Kidstime Foundation", to: "http://www.kidstimefoundation.org") %></strong>
-    is a UK charity supporting children and young people affected by the mental health problems of a parent or sibling</p>
-  </div>
+    <p><strong><%= link "ADHD Social Club", to: "https://www.bexley.gov.uk/events/adhd-social-club", class: "hl-aqua flex-wrap underline" %></strong> for young people
+      aged 8 to 11 years. The ADHD Social Club meets on a weekly basis and provides a safe environment for young people to meet with
+      their peers and have fun. The weekly sessions will provide a range of activities including art, fun games, cooking and other 
+      hands-on activities in a structured way. Parents and carers are asked to fully engage in the sessions to maximise the benefit for the whole family. 
+      The sessions look at developing key social skills, improve concentration, attentiveness and skills for life.  
+      The club is held at North Cray Community Centre, 1 Davis Way, Sidcup, DA14 5JR every Wednesday from 5:30pm to 6:30pm, admission is £3 per session.  
+      For more information, email <%= link "yadt@sky.com", to: "mailto:yadt@sky.com", class: "hl-aqua flex-wrap underline" %>
+   </p>
+
+    <p><strong><%= link "Greenwich and Lewisham Young People’s theatre", to: "http://www.glypt.co.uk/", class: "hl-aqua flex-wrap underline" %></strong> is the
+      home of great theatre and arts activities for young people and families. There’s something for everyone of all ages! Call <%= link "020 8854 1316", to: "tel:02088541316", class: "hl-aqua flex-wrap underline" %> or
+      find us at The Tramshed, 51-53 Woolwich New Road, London SE18 6ES 
+  </p>
+      
+    <p><strong><%= link "National Autistic Society (NAS) - Greenwich Branch", to: "https://nasgreenwich.org.uk/", class: "hl-aqua flex-wrap underline" %></strong> was 
+      started in 1998 by a group of parents of autistic children who wanted to share their knowledge and experience to support other 
+      families connected to Autism Spectrum Conditions (ASCs). The branch has over 400 members with an increasing number of 
+      adults and professionals represented.
```

Comment:
hey @RobStallion I didn't want a new paragraph really so have just added a <br>
Can this whole section, including the line break and list of bullet points be within a paragraph?